UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/AccessibilityStatement/Index.cshtml

```diff
@@ -0,0 +1,49 @@
+        <li><i class="bi bi-square-fill list-icon" aria-hidden="true"></i> Navigate most of the website using speech recognition software.</li>
+        <li><i class="bi bi-square-fill list-icon" aria-hidden="true"></i> Listen to most of the website using a screen reader (including the most recent versions of NVDA and VoiceOver).</li>
+        <li><i class="bi bi-square-fill list-icon" aria-hidden="true"></i> See descriptive attributes for all images.</li>
+        <li><i class="bi bi-square-fill list-icon" aria-hidden="true"></i> See descriptions for all links.</li>
+    </ul>
+
+    <p>We’ve also made the website text as simple as possible to understand.</p>
+    <p><a href="https://abilitynet.org.uk/" target="_blank">AbilityNet</a> has advice on making your device easier to use if you have a disability.</p>
+    <h2>Feedback and contact information</h2>
+    <p>If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille:</p>
+    <ul class="msi-list-style">
+        <li><i class="bi bi-square-fill list-icon" aria-hidden="true"></i> Email <a href="mailto:customerservices@ukho.gov.uk">customerservices@ukho.gov.uk</a></li>
+        <li><i class="bi bi-square-fill list-icon" aria-hidden="true"></i> Call +44 (0)1823 484444</li>
+    </ul>
+
+    <p>We’ll consider your request and reply to you within 5 working days.</p>
+    <h3>Reporting accessibility problems with this website</h3>
+    <p>We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact <a href="mailto:customerservices@ukho.gov.uk">customerservices@ukho.gov.uk</a>to let us know your concerns.</p>
+    <h3>Enforcement procedure</h3>
+    <p>The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, <a href="https://www.equalityadvisoryservice.com/">contact the Equality Advisory and Support Service (EASS).</a>
+    </p>
+    <h3>Technical information about this website’s accessibility</h3>
+    <p>The UK Hydrographic Office is committed to making this website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.</p>
+    <h3>Compliance status</h3>
+    <p>This website is partially compliant with the Web Content Accessibility Guidelines version 2.1 AA standard, due to the non-compliances listed below.</p>
+    <h3>Non-accessible content</h3>
+    <p>The content listed below is non-accessible for the following reasons.</p>
+    <h3>Non-compliance with the accessibility regulations</h3>
+    <p>N/A</p>
```

Comment:
So we're saying that:
> This website is partially compliant...due to the non-compliances listed below.

and:
> The content listed below is non-accessible for the following reasons.

but then we're saying that the non compliant reasons are:
> N/A




Do we have any non compliant content?

Reply:
As mentioned below we do not have non compliant reasons as of now but we are yet to complete accessibility testing for our application, so may have some non-compliant reasons once testing is completed.

So not sure if we should keep this section as it is for now.

Reply:
I think I would comment out (so that it can be easily re-added):
```
    <h3>Compliance status</h3>
    <p>This website is partially compliant with the Web Content Accessibility Guidelines version 2.1 AA standard, due to the non-compliances listed below.</p>
    <h3>Non-accessible content</h3>
    <p>The content listed below is non-accessible for the following reasons.</p>
    <h3>Non-compliance with the accessibility regulations</h3>
    <p>N/A</p>
```

and just put in the following for now:
```
    <h3>Compliance status</h3>
    <p>This website is compliant with the Web Content Accessibility Guidelines version 2.1 AA standard.</p>
```

Reply:
@barrie-cooper  Should we be stating full compliance if we are not necessarily fully compliant?

Given that there is a PR to update these around the corner, perhaps we could leave this as-is for now, or maybe state that in the list section?

Maybe something like:
    ```
    <h3>Non-compliance with the accessibility regulations</h3>
    <p><i>List of non-compliances pending</i></p>
```